Makefile

```diff
@@ -84,12 +84,12 @@ ifeq ($(NOLINT), 0)
 endif
 
 lintall:
-	$(Echo) "Linting /src\n"
+	$(Echo) "Linting src/"
```

Comment:
Can we leave the newlines in there, without them it becomes really hard to read

Reply:
Actually, the original was printed as:
```
Linting /src\n

```
Echo does its newline on its own, check the travis log for how it looks now.